ui/jstests/test_manage_taxonomies.jsx

```diff
@@ -35,39 +35,278 @@ define(['QUnit', 'jquery', 'setup_manage_taxonomies', 'reactaddons',
       }
     ]
   };
-  function reportError(msg) {
-    if (msg) {
-      console.error(msg);
-    }
+  /**
+   * Common assertions for Add terms, use in term and taxonomy
+   * components tests
+   * @param {Object} assert - For assertions
+   * @param {Object} component - Root DOM object in react class
+   * @return {Object} devItems - DOM object of class 'panel-body'
+   * */
+  function  assertAddTermCommon(assert, component) {
+    // check term title
+    var listOfLinks = React.addons.TestUtils.
+      scryRenderedDOMComponentsWithTag(
+        component,
+        'a'
+    );
+    assert.equal(listOfLinks.length, 2);
+    var linkHeader = listOfLinks[0].getDOMNode();
+    assert.equal(linkHeader.innerHTML, vocabulary.name);
+    //test items
+    var devItems = React.addons.TestUtils.
+      findRenderedDOMComponentWithClass(
+        component,
+        'panel-body'
+    );
+    var itemList = React.addons.TestUtils.
+      scryRenderedDOMComponentsWithTag(
+        devItems,
+        'li'
+    );
+    assert.equal(itemList.length, 2);
+    return devItems;
+  }
+  /**
+   * Common assertions for Add Vocabulary, use in term and taxonomy
+   * components tests
+   * @param {Object} assert - For assertions
```

Comment:
`Object` here should be set to `QUnit.assert` described in [their docs](https://api.qunitjs.com/QUnit.assert/)